### PR TITLE
EventEditor: Team Editor

### PR DESCRIPTION
**Pull request by booleanrecep**
### User is able to see bio card of artists/profiles in drop-down list and able to select,edit,delete.
`Demo link:`  https://wedance-booleanrecep-sc-427.netlify.app/events/-/edit
#### Step to follow:
* Go to your profile
* Click  **My Profile** tab
* Click **Add event** button
##### Demo:
sc-427

### CODE REVIEW

File: components/TProfileSelect/TBioCard.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <div
+      :class="[
+        { 'hover:bg-gray-200 cursor-pointer': !showTools },
+        'flex-row gap-2 flex justify-between items-center h-16 m-1 border-gray-200 border-2  px-2',
+      ]"
+    >
+      <div class="flex-shrink-0 w-12">
+        <a href="#" class="block relative">
+          <img
+            alt="profil"
```
Comment (best-lucky1030) [inline components/TProfileSelect/TBioCard.vue:16]: this is typo.
not `profil`, it's `profile`.

**Review by best-lucky1030 — COMMENTED** [review]
Typo.

**Comment by razbakov** [comment]
@booleanrecep please deploy demo on netlify and provide link, so that we can test

**Comment by razbakov** [comment]
@booleanrecep great work! 

Now let's replace hardcoded array of profiles with actual profiles. You can reuse the code of /search page for that. It's using Algolia search.

**Comment by razbakov** [comment]
Nice!

Few changes:

- After adding profile - input should be cleared.
- List should be displayed completely without vertical scroll

And the next thing to do:

- edit bio and role

**Comment by razbakov** [comment]
@booleanrecep please re-use existing TPopup component

**Comment by razbakov** [comment]
@booleanrecep it should not be possible to add the same profile multiple times

**Comment by razbakov** [comment]
@booleanrecep it's not possible to save role

**Review by best-lucky1030 — COMMENTED** [review]
looks good.

**Comment by razbakov** [comment]
Changes:
- rename field to "Guests"
- place after Description
- use label position "top"

**Review by razbakov — CHANGES_REQUESTED** [review]
Changes:
- rename field to "Team"
- place after Description
- use label position "top"

**Pull request closed without merging**